src/openrct2/object/SceneryGroupObject.cpp

```diff
@@ -142,36 +142,36 @@ uint32_t SceneryGroupObject::ReadJsonEntertainerCostumes(const json_t* jCostumes
 
         // For some reason the flags are +4 from the actual costume IDs
         // See staff_get_available_entertainer_costumes
-        costumes |= 1 << (entertainer + 4);
+        costumes |= 1 << (static_cast<uint8_t>(entertainer) + 4);
```

Comment:
This is weirdle/oddly familiar to the calculation being done on `EntertainerCostumeToSprite` 🤔

Reply:
Done!